packages/scratch-vm/src/extensions/scratch3_musiccreation/index.js

```diff
@@ -568,14 +568,24 @@ class Scratch3MusicCreation {
      */
     async _getAndPlayMagentaNotes(RNN, args, utils, inst) {
         let magenta_notes = null;
+        let valid = true;
         if (RNN) {
-            magenta_notes = await this.musicAccompanimentHelper.testMagentaRNN(this.noteList, args, utils);
+            if (this.noteList.length > 0) {
+                const low = 48;
+                const hi = 83;
+                const filtered = this.noteList.filter(note => note[0] >= low && note[0] <= hi);
```

Comment:
We discussed trying to cast notes up or down octaves so that we can preserve the notes